.node-version

```diff
@@ -1 +1 @@
-12.16.1
+12.16.3
```

Comment:
can we just bump to version 14?

Reply:
I don't know if Vercel is running 14 yet, so we would be running locally on a different major version than in production

Reply:
@csell5 The silly thing is that we completely forgot to change the "engines" parameter in package.json, so it's actually still running on Node 10.